packages/eas-cli/src/submissions/archive-source/ArchiveFileSource.ts

```diff
@@ -0,0 +1,284 @@
+import chalk from 'chalk';
+import { URL, parse as parseUrl } from 'url';
+import { validate as uuidValidate } from 'uuid';
```

Comment:
```suggestion
import { validate as validateUuid } from 'uuid';
```
or 
```suggestion
import * as uuid from 'uuid';
```
or just 
```suggestion
import uuid from 'uuid';
```
if that works.